custom_components/apple_tv/__init__.py

```diff
@@ -151,6 +153,8 @@ def __init__(self, hass, config_entry):
         self.message = None
         self.atv = None
         self._is_on = not config_entry.options.get(CONF_START_OFF, False)
+        self._is_pwr_mgmt_on = config_entry.options.get(CONF_PWR_MGMT, False)
```

Comment:
This is a bit tricky. Doing it like this force the user to restart Home Assistant in order for the option to take effect. One way is to read this value from `self.config_entry` every time. But since config entries supports listeners, we should listen for updates and update variable accordingly.

https://github.com/home-assistant/core/blob/dev/homeassistant/config_entries.py#L381